packages/assets-controllers/src/TokenBalancesController.ts

```diff
@@ -119,7 +117,9 @@ export type AllowedEvents =
   | TokensControllerStateChangeEvent
   | PreferencesControllerStateChangeEvent
   | NetworkControllerStateChangeEvent
-  | KeyringControllerAccountRemovedEvent;
+  | KeyringControllerAccountRemovedEvent
+  | AccountActivityServiceBalanceUpdatedEvent
```

Comment:
Heads up that updates to `AllowedEvents` in a package are breaking changes (because the allowlist needs to be updated on the client side). Can you mention this in the changelog?